### PR TITLE
[ISSUE-567] Generate changelog action is configured to use custom labels

### PR DESCRIPTION
**Pull request by misanthropicat**
Signed-off-by: misanthropicat <misanthr0picat@gmail.com>

### Issue dell/csi-baremetal#567

Generate changelog action is configured to use PR labels

## PR checklist
- [x] Add link to the issue
- [x] Choose PR label
- [ ] New unit tests added
- [ ] Modified code has meaningful comments
- [ ] All TODOs are linked with the issues
- [ ] All comments are resolved

## Testing
On fork repo:
[Workflow run in csi repo](https://github.com/misanthropicat/csi-baremetal/actions/runs/1363852984)
[Workflow in operator repo](https://github.com/misanthropicat/csi-baremetal-operator/actions/runs/1363854792)
image

### CODE REVIEW

**Comment by mishoyama** [comment]
@misanthropicat if we use another label or label is not set will it work?

**Review by yurkov-anton — APPROVED** [review]

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -140,9 +140,8 @@ jobs:
         id: commit_docs
         uses: stefanzweifel/git-auto-commit-action@v4
         with:
-          commit_message: "[issue#${{ github.event.inputs.issue_number }}] Update Helm repo index"
+          commit_message: "Update Helm repo index"
           commit_options: --amend --signoff
```
Comment (mishoyama) [inline .github/workflows/release.yml:144]: `--amend` will add your changes to the existing commit. Please remove it.

**Review by mishoyama — CHANGES_REQUESTED** [review]

File: .github/workflows/release.yml (hunk 1 above)
Comment (misanthropicat) [reply .github/workflows/release.yml]: Done

**Review by rkiselenko — APPROVED** [review]
LGTM

**Pull request merged**